frontend/src/Square.js

```diff
@@ -0,0 +1,49 @@
+import {is_defined, get_position} from './App'
+import React from 'react';
+import axios from 'axios';
+class Square extends React.Component {
+  constructor(props){
+    super(props);
+    const value = this.props.value
+
+   }
+  handle_click(){
+    console.log(this.props.value)
+    const parameter = {"data" : this.props.value}
+    axios.post('http://localhost:5000/pick',parameter).then(
+        response => {
+            this.props.handle_render()
+        })
+
+  }
+
+  render() {
+    return (
+        <button
+            className="square"
+            style = {{color: this.props.value.color,
+            background: square_background(
+                this.props.row,this.props.column)}}
+            id = {this.props.id}
+            onClick={() =>this.handle_click()}
+           >
+            {this.props.color}
+        </button>
+    );
+  }
+
+}
+
+
+function square_background(row,column){
+    if(is_defined(row) & is_defined(column)){
```

Comment:
you can also use lodash _.isNumber()

Reply:
i just looked at the lodash doc there will probably be more things that i can use it for, thanks